networkx/algorithms/traversal/breadth_first_search.py

```diff
@@ -41,24 +45,39 @@ def bfs_edges(G, source, reverse=False):
     Based on http://www.ics.uci.edu/~eppstein/PADS/BFS.py
     by D. Eppstein, July 2004.
     """
+    if depth_limit is not None and depth_limit < 0:
```

Comment:
This line is not necessary. It will be caught by the check for `current_depth <= depth_limit` below.